app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -116,6 +117,8 @@ public View inflateLayout(LayoutInflater inflater, @Nullable ViewGroup container
         menuView = view.findViewById(R.id.menu);
         menuView.setOnClickListener(this);
 
+        new KeyboardUtil(getActivity(), view);
```

Comment:
This is a super weird API. We create an object that isn't even needed. It looks like this should have been a static helper method.